app/views/videos/show.html.haml

```diff
@@ -12,41 +11,49 @@
           - if @video.average_rating != nil
             %span Rating: #{@video.average_rating}/5.0
           <hr>
-          = @video.description
+          %p= @video.description
+
+          .actions
+            %a.btn.btn-primary(href="") Watch Now
+            = link_to '+ My Queue', queue_items_path(video_id: @video.id), method: 'post', class:'btn btn-default'
+
+
+
 %section.reviews.container
   .row
     .col-sm-10.col-sm-offset-1
       = form_for [@video,@review] do |f|
-        = f.label :title
-        = f.text_field :title
-
-        = f.label :rating
-        = f.select(:rating, options_for_select([5,4,3,2,1].map{|num| pluralize(num, 'Star')}))
+        .form-group
+          = f.label "Rate this video"
+          .row
+            .col-sm-3
+              = f.select(:rating, options_for_select([5,4,3,2,1].map{|num| pluralize(num, 'Star')}))
```

Comment:
You can eventually move `options_for_select([5,4,3,2,1].map{|num| pluralize(num, 'Star')})` to a helper method and reuse this logic for rating in `queue_items#index` view